### PR TITLE
Another 10 MH3 cards

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Accursed Marauder
+ManaCost:1 B
+Types:Creature Zombie Warrior
+PT:2/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, each player sacrifices a nontoken creature.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Player | SacValid$ Creature.nonToken
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt:6]: SacMessage$ nontoken creature

File: forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Accursed Marauder
+ManaCost:1 B
+Types:Creature Zombie Warrior
+PT:2/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, each player sacrifices a nontoken creature.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Player | SacValid$ Creature.nonToken
+SVar:NeedsToPlay:Creature.nonToken+OppCtrl
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt:7]: DeckHas:Ability$Sacrifice

File: forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Breathe Your Last
+ManaCost:1 B B
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | SubAbility$ DBGainLife | SpellDescription$ Destroy target creature or planeswalker. You gain 1 life for each of its colors.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Targeted$CardNumColors
+Oracle:Destroy target creature or planeswalker. You gain 1 life for each of its colors.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt:7]: DeckHas:Ability$LifeGain

File: forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Consuming Corruption
+ManaCost:B B
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBGainLife | SpellDescription$ CARDNAME deals X damage to target creature or planeswalker and you gain X life, where X is the number of Swamps you control.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Count$TypeYouCtrl.Swamp
+Oracle:Consuming Corruption deals X damage to target creature or planeswalker and you gain X life, where X is the number of Swamps you control.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt:7]: DeckHas:Ability$LifeGain

File: forge-gui/res/cardsfolder/upcoming/dreadmobile.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Dreadmobile
+ManaCost:2 B
+Types:Artifact Vehicle
+PT:3/3
+K:Menace
+A:AB$ PutCounter | Cost$ 1 Sac<1/Artifact.Other;Creature.Other/another artifact or creature> | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on CARDNAME.
+K:Crew:1
+Oracle:Menace\n{1}, Sacrifice another artifact or creature: Put a +1/+1 counter on Dreadmobile.\nCrew 1 (Tap any number of creatures you control with total power 1 or more: This Vehicle becomes an artifact creature until end of turn.)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/dreadmobile.txt:8]: DeckHas:Ability$Sacrifice|Counters

File: forge-gui/res/cardsfolder/upcoming/dreamdrinker_vampire.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Name:Dreamdrinker Vampire
+ManaCost:1 B
+Types:Creature Vampire
+PT:2/1
+K:Lifelink
+K:Adapt:1:1 B
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigPump | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME, it gains menace until end of turn.
+SVar:TrigPump:DB$ Pump | Defined$ Self | KW$ Menace
+DeckHas:Ability$Counters
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/dreamdrinker_vampire.txt:9]: `|LifeGain`

File: forge-gui/res/cardsfolder/upcoming/gravedig.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Gravedig
+ManaCost:1 B
+Types:Sorcery
+A:SP$ Charm | Choices$ DBToken,DBChangeZone | Defined$ You | CharmNum$ 1
+SVar:DBToken:DB$ Token | ValidTgts$ Player | TgtPrompt$ Select target player. | TokenAmount$ 1 | TokenScript$ b_2_2_zombie | TokenOwner$ TargetedPlayer | StackDescription$ SpellDescription | SpellDescription$ Target player creates a 2/2 black Zombie creature token.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature in your graveyard | SpellDescription$ Return target creature card from your graveyard to your hand.
+K:Entwine:2
+DeckHas:Ability$Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/gravedig.txt:8]: `|Graveyard & Type$Zombie`

File: forge-gui/res/cardsfolder/upcoming/unfathomable_truths.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+Name:Unfathomable Truths
+ManaCost:4 U
+Types:Instant
+K:Devoid
+A:SP$ Draw | NumCards$ 3 | SpellDescription$ Draw three cards and create a 0/1 colorless Eldrazi Spawn creature token with "Sacrifice this creature: Add {C}." | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_0_1_eldrazi_spawn_sac
+Oracle:Devoid (This card has no color.)\nDraw three cards and create a 0/1 colorless Eldrazi Spawn creature token with "Sacrifice this creature: Add {C}."
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/unfathomable_truths.txt:7]: DeckHas:Ability$Token|Sacrifice & Type$Eldrazi|Spawn

File: forge-gui/res/cardsfolder/upcoming/gravedig.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Name:Gravedig
+ManaCost:1 B
+Types:Sorcery
+A:SP$ Charm | Choices$ DBToken,DBChangeZone | Defined$ You | CharmNum$ 1
+SVar:DBToken:DB$ Token | ValidTgts$ Player | TgtPrompt$ Select target player. | TokenAmount$ 1 | TokenScript$ b_2_2_zombie | TokenOwner$ TargetedPlayer | StackDescription$ SpellDescription | SpellDescription$ Target player creates a 2/2 black Zombie creature token.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/gravedig.txt:5]: just remove this TgtPrompt I think

File: forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt [hunk 10]
```diff
@@ -0,0 +1,7 @@
+Name:Consuming Corruption
+ManaCost:B B
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBGainLife | SpellDescription$ CARDNAME deals X damage to target creature or planeswalker and you gain X life, where X is the number of Swamps you control.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Count$TypeYouCtrl.Swamp
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt:6]: Should this be `SVar:X:Count$Valid Swamp.YouCtrl`?

Tendrils of Corruption also uses this syntax

Probably both are ok?

File: forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt [hunk 11]
```diff
@@ -0,0 +1,7 @@
+Name:Breathe Your Last
+ManaCost:1 B B
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | SubAbility$ DBGainLife | SpellDescription$ Destroy target creature or planeswalker. You gain 1 life for each of its colors.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ X
+SVar:X:Targeted$CardNumColors
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt:6]: This is good enough, right? Don't need any extra LKI stuff

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breathe_your_last.txt:6]: yes

File: forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt (hunk 10 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/consuming_corruption.txt:6]: yea a bit older variant
maybe one day we remove it?

File: forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt (hunk 2 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/accursed_marauder.txt:7]: I think this AI hint would still be good?

**Review by Northmoc — APPROVED** [review]
👍

**Pull request merged**